installation/windows/toolbox.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Toolbox Extension for VS
+page_title: Telerik Toolbox VS Extension for .NET MAUI on Windows
+description: Learn more about the Telerik Toolbox extension for .NET MAUI on Windows and how to use the toolbox with the Telerik UI for .NET MAUI library.
+slug: toolbox-support
+position: 3
+previous_url: /installation/windows/toolbox-support
+---
+
+# Telerik Toolbox for .NET MAUI on Windows
+
+Telerik products support the Toolbox extension for Visual Studio (VS) on Windows. The Toolbox is part of the Visual Studio extensions for Telerik UI for .NET MAUI and facilitates the process of adding Telerik controls to your .NET MAUI application.
+
+The Telerik UI for .NET MAUI VS extensions are distributed with the Telerik UI for .NET MAUI [MSI installation]({%slug maui-getting-started%}). You can also download and install them as a separate product from the [Visual Studio Marketplace](https://marketplace.visualstudio.com/) for Visual Studio 2022.
+
+> For more information, go to the topic about [getting started with the Telerik VS extensions]({%slug visualstudio-extensions%}) topic.
+
+## Adding Controls with the Toolbox
+
+To add the desired Telerik UI for .NET MAUI controls with the Toolbox:
+
+1. In Visual Studio, navigate to the **Extensions** tab.
+1. Select **Telerik** > **Telerik UI for .NET MAUI** > **Open Telerik UI for .NET MAUI Toolbox**.
+
```

Comment:
To prevent the incorrect rendering of the list, add one more space character to all lines that aren't list items.